### ISSUE TITLE
Write the stub method anyBlock()

### ISSUE THREAD

**Issue by michalcohen** [TechnionYP5777/IntelliJ-Spartanizer#98]
A method that will be used to indicate that it doesn't metter what a block contains. It shoult be parsed by TreeTipperBuilder, Prunned correctly to the new PsiElement you will create "GenericPsiBlock", matched with the matcher and replaced correctly by the replacer.
Fill free to talk with me, @orenafek or @AnnaBel7 :)

**Comment by amirsagiv83** [comment]
Mission accomplished. an explanation and description of our work will be uploaded later, and then issue will be closed. @RoeyMaor  :confetti_ball: :smile:  :pig:  :tada:

**Comment by RoeyMaor** [comment]
![image](https://cloud.githubusercontent.com/assets/16507852/22158469/030b1ee4-df45-11e6-9d6f-ec51733b75a6.png)

![image](https://cloud.githubusercontent.com/assets/16507852/22158562/7b9cbd2c-df45-11e6-85f5-aa7e6e181e15.png)

🐷 @amirsagiv83 🔝

**Comment by amirsagiv83** [comment]
Just to explain the above image and close this issue.
We added the anyBlock stub method which presents any block of statements possible. it can be a single statement or a whole block (surrounded by curly braces). we also added a GenericPsiBlock to replace the stub method in the pruning phase and we updated the iz.conforms method to work with anyblock type.
Also, We added a small check to the matcher so it won't try to compare the content of a block if it's an anyblock.

And, there results are right above this comment. it works fantastically, and this stub method should be useful to lots of tips.
 🎉 🎈 🎊
